### DISCUSSION TITLE
Only specific elements editable

### DISCUSSION THREAD

**Discussion by EstrategyTeam** [GrapesJS/grapesjs#6336] [Q&A]
Is it possible to prevent the whole body from getting editable, except specific elements on the page?
Our goal is to create one base template, which can be used for multiple pages, this page contains specific static/fixed elements.
The header, footer and some elements on the page shouldn't be changed.

Is it possible to only make block3 editable?
See example below:
```html
<body>
    <header>
        static header (non-editable)
    </header>
    <div class="block1">non-editable</div>
    <div class="block2">non-editable</div>
    <div class="block3">editable</div>
    <div class="block4">non-editable</div>
    <footer>
        static footer (non-editable)
    </footer>
</body>
``` 

The header and footer can contain many child elements, which we also don't want to editable.

**Comment by artf** [comment]
Check the examples here: https://github.com/GrapesJS/grapesjs/issues/3091#issuecomment-1185359751
